generic/tests/cfg/iometer_windows.cfg

```diff
@@ -73,3 +73,37 @@
                     bg_mode = yes
                 - after_test:
                     bg_mode = no
+        - with_multi_scsi_disks:
+            only virtio_scsi
+            run_timeout = 3600
+            icf_name = "iometer_multi_disks.icf"
+            format_multi_disks = yes
+            disk_indexes = "1 2 3 4 5 6"
+            disk_letters = "I J K L M N"
+            disk_fstypes = "fat32 fat ntfs fat32 fat ntfs"
```

Comment:
Hi yhong,

The detail file system type support status for windows os as follows:
1. On Win2016-64, Win2019-64: NTFS, FAT32, ReFS, test 3 disks is enough.
2. On Win2012-64, Win2012-r2: NTFS, FAT, FAT32, ReFS, need test 4 disks.
3. On others, Win7-32/64, Win8-32/64, Win8.1-32/64, Win10-32/64, Win2008-32/64/r2: NTFS, FAT32, only 2 disks is enough.

Please help to re-configure the cfg to match the os supported status in actually. Thanks~

Reply:
Hi yhong, Update the detail FS support status on windows:

1. On win2012-64, win2012-r2, win2016-64, win2019-64, supported file system type as: FAT, FAT32, NTFS, ReFS.
2. On Win7-32/64, Win8-32/64, Win8.1-32/64, Win10-32/64, Win2008-32/64/r2, supported file system type as: FAT, FAT32, NTFS.

Note: FAT system does support on  <=3G partition, ReFS does support on  >=1.5G partition, FAT32 does support on <=128G partition, and NTFS does support on <=2T partition.

So, suggest tested with 2G disk for Refs file system, others file system keep 1G also is ok.

Best Regrds~
Peixiu

Reply:
> Hi yhong, Update the detail FS support status on windows:
> 
>     1. On win2012-64, win2012-r2, win2016-64, win2019-64, supported file system type as: FAT, FAT32, NTFS, ReFS.
> 
>     2. On Win7-32/64, Win8-32/64, Win8.1-32/64, Win10-32/64, Win2008-32/64/r2, supported file system type as: FAT, FAT32, NTFS.
> 
> 
> Note: FAT system does support on <=3G partition, ReFS does support on >=1.5G partition, FAT32 does support on <=128G partition, and NTFS does support on <=2T partition.
> 
> So, suggest tested with 2G disk for Refs file system, others file system keep 1G also is ok.
> 
> Best Regrds~
> Peixiu

Hi @peixiu  thanks for you comments. it's great help to me.